src/internal/operators/shareReplay.ts

```diff
@@ -141,7 +141,6 @@ export function shareReplay<T>(bufferSize?: number, windowTime?: number, schedul
  *
  * @see {@link publish}
```

Comment:
This one was removed, can you remove it from here as well?

Reply:
Done.